### PR TITLE
Add config to example client RPC

### PR DESCRIPTION
**Pull request by rnicoll**

### CODE REVIEW

File: kotlin-source/src/main/kotlin/com/example/client/ExampleClientRPC.kt [hunk 1]
```diff
@@ -25,7 +26,7 @@ private class ExampleClientRPC {
     fun main(args: Array<String>) {
         require(args.size == 1) { "Usage: ExampleClientRPC <node address>" }
         val nodeAddress = HostAndPort.fromString(args[0])
-        val client = CordaRPCClient(nodeAddress)
+        val client = CordaRPCClient(nodeAddress, configureTestSSL(), null)
```
Comment (Clintonio) [inline kotlin-source/src/main/kotlin/com/example/client/ExampleClientRPC.kt]: Can we drop the optional null parameter here? It adds no useful information.

File: java-source/src/main/java/com/example/client/ExampleClientRPC.java [hunk 2]
```diff
@@ -27,7 +27,7 @@ public static void main(String[] args) throws ActiveMQException, InterruptedExce
 
         final Logger logger = LoggerFactory.getLogger(ExampleClientRPC.class);
         final HostAndPort nodeAddress = HostAndPort.fromString(args[0]);
-        final CordaRPCClient client = new CordaRPCClient(nodeAddress, null, null);
+        final CordaRPCClient client = new CordaRPCClient(nodeAddress, ConfigUtilities.configureTestSSL(), null);
```
Comment (Clintonio) [inline java-source/src/main/java/com/example/client/ExampleClientRPC.java]: Similarly can we drop the null parameter here?

Comment (rnicoll) [inline java-source/src/main/java/com/example/client/ExampleClientRPC.java]: Not in Java without changing the how the Corda class declaration is written, as it complains bitterly right now.

**Comment by rnicoll** [comment]
Removed null from Kotlin version as requested

**Comment by rnicoll** [comment]
@Clintonio is this okay to merge now?

**Comment by rnicoll** [comment]
Unfortunately JvmOverloads doesn't seem to affect classes.

**Pull request merged**